operations/System_Recovery.md

```diff
@@ -574,4 +576,57 @@ The latest CSM documentation has been installed on the master nodes. See [Check
 
    See [Nexus Export and Restore](package_repository_management/Nexus_Export_and_Restore.md)
 
+### Slurm
+
+To restore Slurm data from backup, follow sections
+*10.3.11 Restore Slurm Accounting Database from Backup* and
+*10.3.13 Restore Slurm Spool Directory from Backup* in the
+**HPE Cray Programming Environment Installation Guide: CSM on HPE Cray EX Systems (S-8003)**.
+
+After restoring Slurm data from backup, check that the procedure was successful.
+
+1. (`uan#`) Check the accounting records.
+
+   ```bash
+   sacct -S <date>
```

Comment:
Would `sacct -a` be a simpler check?  How would one determine the <date> to use if checking foraccount starttimes? 
Also is seems that the <date> is based on CDT+12??  Maybe for those the use slurm this is all a known - so disregard if you feel customers will know how to select a valid <date> for this check.
```
uan01:~ # date
Fri 31 Mar 2023 05:52:54 PM CDT
uan01:~ # sacct -S 2023-03-31T17:55:00
sacct: error: Start time (2023-03-31T17:55:00) requested is after end time (2023-03-31T17:53:15).
uan01:~ # date
Fri 31 Mar 2023 05:53:17 PM CDT
uan01:~ # sacct -S 2023-03-31T17:53:00
JobID           JobName  Partition    Account  AllocCPUS      State ExitCode
------------ ---------- ---------- ---------- ---------- ---------- --------
692208         msgget01      workq                    32    RUNNING      0:0
```

Reply:
`sacct -a` will only show the jobs run during the current day. If the backup was done more than a day before the restore, it won't show any jobs. To make sure the backup/restore worked, you want to show jobs from before the backup was made.

Reply:
would it make sense to then say that the <date> should be from before the backup was taken?  And what TZ is that date based on?